File: src/BaekJoon/C++/baekjoon_1546.cpp
// https://www.acmicpc.net/problem/1546

#include <iostream>
using namespace std;

double score[1000];

int main() {
	int n;
	cin >> n;

	double avg = 0.0, max = 0.0;

	for(int i = 0; i < n; i++) {
		cin >> score[i];
		if(score[i] > max)
			max = score[i];
	}

	for(int i = 0; i < n; i++) {
		score[i] = (double)(score[i] / max) * 100.0;
		avg += score[i];
	}

	avg /= n;
	printf("%.2f\n", avg);

	return 0;
}

File: src/BaekJoon/Java/baekjoon_1546.java
package BaekJoon.Java;
// https://www.acmicpc.net/problem/1495

import java.util.Scanner;

public class baekjoon_1546 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		double[] score = new double[n];
		double avg = 0.0;
		double max = 0.0;

		for(int i = 0; i < score.length; i++) {
			score[i] = sc.nextInt();

			if(score[i] > max)
				max = score[i];
		}

		for(int i = 0; i < score.length; i++) {
			score[i] = (double)(score[i] / max) * 100.0;
			avg += score[i];
		}

		avg /= n;

		System.out.printf("%.2f\n", avg);

		sc.close();
	}
}